triggers/display_action_list.ts

```diff
@@ -0,0 +1,23 @@
+import { Trigger } from "deno-slack-api/types.ts";
+import CheckUser from "../workflows/check_user_action_list.ts";
+import { TriggerContextData, TriggerTypes } from "deno-slack-api/mod.ts";
+
+/**
+ * This link trigger prompts the display action workflow.
+ */
+const startMeetingTrigger: Trigger<typeof CheckUser.definition> = {
```

Comment:
this also seems very generic and similar to the startMeeting workflow
should be changed to like displayUserActionItemsTrigger or something